### DISCUSSION TITLE
react-bulma-components maintenance

### DISCUSSION THREAD

**Discussion by opya** [couds/react-bulma-components#421] [General]
Hi guys,

I was wondering if this repo is still maintained and active because the last commit is over a year ago.

**Comment by ollycrossley** [comment]
By the looks of things, not really. I think it stopped being updated past 0.9.2 and we are now on 0.9.4. 
Unfortunate really but it makes it difficult using this library going forward

**Comment by opya** [comment]
Thank you @ollycrossley.
